### PR TITLE
Appveyor for Windows test unit

### PR DESCRIPTION
**Pull request by jaumard**
Here is the basic configuration for appveyor, it run unit test on x86 and x64 with latest LTS and Stable version of Node.js.
If you need some help for Appveyor configuration with github let me know.
Related to https://github.com/trailsjs/trails/issues/110

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "server"
   ],
   "scripts": {
-    "test": "eslint . && mocha"
+    "test": "eslint --fix . && mocha"
```
Comment (tjwebb) [inline package.json]: `--fix` should be run separately.

Comment (jaumard) [inline package.json]: Ok I remove it :)

**Comment by jaumard** [comment]
Yeah all green ;)

**Comment by Alaneor** [comment]
May I suggest adding a badge to README? Especially the green one! 👏 🎉

**Comment by jaumard** [comment]
Good idea @Alaneor but I don't have url to Trails AppVeyor badges. Only @tjwebb have access to those (on project settings under AppVeyor) they use some random url so I can't guess ^^

**Comment by tjwebb** [comment]
Awesome! I think you win https://www.bountysource.com/issues/31897782-setup-travisci-to-run-tests-against-windows :)

**Comment by tjwebb** [comment]
> Only @tjwebb have access to those (on project settings under AppVeyor) they use some random url so I can't guess ^^

@jaumard I can add you as a user to the AppVeyor project.

**Comment by jaumard** [comment]
Yeah Thanks ! :) no problem to be part of the AppVeyor project if you want. I will add the badge on Readme on this PR like this ;)

**Comment by tjwebb** [comment]
@jaumard I think you need to claim the Bounty in order for me to award it to you.

**Comment by jaumard** [comment]
@tjwebb yes that what I read but I can claim it only when issue was close :)

**Pull request merged**